<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminInterface</class>
 <widget class="QWidget" name="AdminInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AdminInterface{	
	background-color: rgb(198, 245, 255);
}

QTableWidget{	
	background-color: rgba(252, 255, 189,170);
	border: 5px solid rgb(255, 110, 255);
	border-color: rgb(255, 123, 169);
	border-radius: 18px;
}

QPushButton{
	background-color: rgb(255, 229, 235);
	border: 3px solid rgb(255, 125, 153);
    border-radius: 12px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
	border: 3px solid rgb(255, 174, 192);
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

QTabBar::tab {    
	background-color: rgb(170, 244, 220);
    border: 3px solid rgb(255, 202, 206);
    border-bottom-color: rgb(255, 202, 206); /* same as the pane color */
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
	border-bottom-left-radius: 8px;
	border-bottom-right-radius: 8px;
    padding: 2px;
	min-height: 30px;
	min-width : 60px;
}

QTabBar::tab:selected, QTabBar::tab:hover {		
	background-color: rgb(212, 255, 241);
}

QTabBar::tab:selected {	
	background-color: rgb(72, 221, 152);
	border: 3px solid rgb(255, 143, 145);
    border-bottom-color: rgb(255, 143, 145); /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 1px; /* make non-selected tabs look smaller */
}

QTableView::item:selected{
    selection-background-color: rgba(0, 0, 0, 0);
}

QTabWidget:pane{
	border-top:  0px hidden rgba(0,0,0,0);
	background:  transparent; 
}

/*滚动条样式*/

/*****滚动条样式*****************************************************************************/

QScrollBar:vertical{

margin:16px 0px 16px 0px;

background-color:rgba(0,0,0,0);

/*border:0px;*/
border: 2px solid rgb(255, 110, 255);
border-color: rgba(179, 226, 205, 150);
background-color: rgba(176, 216, 203, 90);
border-radius: 8px;
width:17px;
}

QScrollBar::handle:vertical{

background-color: rgba(129, 168, 130, 170);
border-radius:6px;

width:13px;

}

QScrollBar::handle:vertical:hover{
background-color:rgba(111, 152, 133,180);
}

QScrollBar::sub-line:vertical{

subcontrol-position:top;

subcontrol-origin:margin;

background-color:rgba(0,0,0,0);

border:0px hidden rgba(0,0,0,0);

height:0px;

}

QScrollBar::add-line:vertical{

subcontrol-position:bottom;

subcontrol-origin:margin;

background-color:rgba(0,0,0,0);

border:0px hidden rgba(0,0,0,0);

height:0px;

}

/*
QScrollBar::up-arrow:vertical{

border-image:url(:/commonImg/Resources/image/common/arrowUp.png);

width:12px;

height:6px;

}

QScrollBar::down-arrow:vertical{

border-image:url(:/commonImg/Resources/image/common/arrowDown.png);

width:12px;

height:6px;

}
*/

QScrollBar::sub-page:vertical,QScrollBar::add-page:vertical{

background-color:rgba(0,0,0,0);

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="16">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <family>楷体</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>饮品</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QTableWidget" name="tableWidget_drink">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>0</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>188</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>0</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>340</number>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <row>
            <property name="text">
             <string>新建行</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>新建行</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>新建行</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>新建列</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>新建列</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>新建列</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>新建列</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>新建列</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>水果</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget_fruit">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>0</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>188</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>0</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>340</number>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <row>
            <property name="text">
             <string>新建行</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>新建行</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>新建列</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>新建列</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>新建列</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>新建列</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>新建列</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2,5,2,2">
       <item>
        <widget class="QPushButton" name="pushButton_quit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_warning">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>确认补货</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
